dktk/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+# TODO: Is there a reason why the docker-compose files are not at root level? e.g. docker-compose.(dktk|gbn|c4).yml

```

Comment:
My idea for the folder structure is, one project has a folder with docker-compose file and a empty configuration file for the site-conf(Maybe other files for the project as well). With the transition to the multi-bk we can start each project with the same file and can store it at the root level